### PR TITLE
Deploy to Netlify with redirect support

### LINKED ISSUE #45: Add a redirects file so routing / deeplinking works
## Motivation

The redirects file will soon be available in IPFS gateways.

In the meantime, this should also deploy seamlessly to modern static front ends like Netlify, Vercel (that's issue #9 for the full potential list over time).

Adding the redirects file will mean that people attempting to use those platforms will "just work".

## Finish Line

- [ ] Add a `_redirects` file that will enable the template app to work on Cloudflare Pages, Netlify, Vercel, etc
- [ ] Document how to deploy in the README

### PR DESCRIPTION
**Pull request by jessmartin**
# Description

Need to support redirects in order to support deep linking during device linking.

Need to remove the files and details around Fission deployment, since these won't work until redirect support lands.

## Link to issue

Closes #45 

## Type of change

- [x] New feature (non-breaking change that adds functionality)
- [x] This change requires a documentation update
- [x] Comments have been added/updated

### CODE REVIEW

**Comment by jessmartin** [comment]
I can't see any reason to keep Vercel connected. @bgins @therealjeffg y'all ok with me disconnecting it?

**Comment by bgins** [comment]
> I can't see any reason to keep Vercel connected. @bgins @therealjeffg y'all ok with me disconnecting it?

Yep, agreed! Go ahead and disconnect it. 👌

**Review by bgins — APPROVED** [review]
Wonderful! The redirects are working perfectly.

One detail, the favicon isn't loading properly, and I'm seeing some 502 bad gateway errors in the console. Is there some configuration we need for static files on Netlify?

**Comment by jessmartin** [comment]
> One detail, the favicon isn't loading properly, and I'm seeing some 502 bad gateway errors in the console. Is there some configuration we need for static files on Netlify?

Hmmm... interesting. Both redirects _and_ static files are working on `main` without any of the Netlify-specific configuration. I know that Netlify has a default set of redirect rules, so apparently they cover the necessary cases? I'm going to test this out. This may turn out to be a small PR. 😄

File: svelte.config.js [hunk 1]
```diff
@@ -3,17 +3,11 @@ import adapter from '@sveltejs/adapter-static'
...
   preprocess: preprocess({
     postcss: true,
   }),
   kit: {
-    adapter: adapter({
-      pages: 'build',
-      assets: 'build',
-      fallback: null
-    }),
+    adapter: adapter(),
```
Comment (jessmartin) [inline svelte.config.js:10]: The provided options were the defaults, so I simply removed them.

**Pull request merged**